db/migrate/20151435234636_add_container_node_max_container_group.rb

```diff
@@ -0,0 +1,5 @@
+class AddContainerNodeMaxContainerGroup < ActiveRecord::Migration
+  def change
+    add_column :container_nodes, :max_container_groups, :string
```

Comment:
@oschreib as other `capacities` should go through `MiqIECUnits.string_to_value` and be stored as int (no need for a bigint for a while I hope :smile: )